src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,29 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
 
+        Figure[] array = new Figure[6];
+        FigureSupplier supplier = new FigureSupplier();
+
+        for (int i = 0; i < array.length; i++) {
+            if (i <= array.length / 2) {
+                array[i] = supplier.getRandomFigure();
+            } else {
+                array[i] = supplier.getDefaultFigure();
+            }
+        }
+
+        for (int i = 0; i < array.length / 2; i++) {
+            array[i] = supplier.getRandomFigure();
+        }
+
+        for (int i = array.length / 2; i < array.length; i++) {
+            array[i] = supplier.getDefaultFigure();
+        }
+        for (Figure figure : array) {
+            figure.getArea();
+            figure.draw();
+        }
```

Comment:
1) result of getArea() method is ignored so no sense to call it
2) call of draw() method mey be included into first cycle
```suggestion
```